dashboard/test/ui/features/foundations/user_menu.feature

```diff
@@ -7,6 +7,18 @@ Scenario: Signed Out - sign in button shows
   And I wait until element "#signin_button" is visible
   And I wait until element ".display_name" is not visible
 
+Scenario: Signed Out - create account button shows on code.org page
```

Comment:
Should these also be `no mobile`?

Reply:
`@no_mobile` is set at the very top on line 1.